src/createAnimatedComponent.js

```diff
@@ -344,6 +355,12 @@ export default function createAnimatedComponent(Component) {
       return props;
     }
 
+    _obtainViewName() {
+      const viewConf =
+        this._component?.viewConfig || this._component?.root?.viewConfig;
+      return viewConf.uiViewClassName;
```

Comment:
should be `return viewConf?.uiViewClassName;`, if `viewConf` is `undefined` it will break

Reply:
fixed 6609d065d654dd05e644bee42dfdb62405907c7e